src/lavinmq/clustering/controller.cr

```diff
@@ -74,7 +80,7 @@ class LavinMQ::Clustering::Controller
           break
         end
       end
-      repli_client = r = Clustering::Client.new(@config, @id, secret)
+      @repli_client = repli_client = r = Clustering::Client.new(@config, @id, secret)
       spawn r.follow(uri), name: "Clustering client #{uri}"
```

Comment:
```suggestion
      @repli_client = repli_client = Clustering::Client.new(@config, @id, secret)
      spawn repli_client.follow(uri), name: "Clustering client #{uri}"
```

Reply:
This won't work because `repli_client` is nilable, so had to keep the `r`.